aten/src/ATen/native/native_functions.yaml

```diff
@@ -717,6 +717,8 @@
 - func: atleast_3d.Sequence(Tensor[] tensors) -> Tensor[]
   variants: function
 
+- func: attn(Tensor q, Tensor k, Tensor v) -> (Tensor, Tensor)
```

Comment:
Note that if you actually put some names for the output Tensors, it will automatically become a named tuple in python. Which makes it much nicer!

Reply:
neat!